ui/src/pages/Logs/Logs.tsx

```diff
@@ -160,7 +209,13 @@ export default function Logs() {
             </TableHead>
             <TableBody>
               {filteredLogs.map((row) => (
-                <Row {...row} />
+                <Row
+                  {...row}
+                  containerLabelColor={containerLabelColor}
+                  setContainerLabelColor={setContainerLabelColor}
+                  containerIconColor={containerIconColor}
+                  setContainerIconColor={setContainerIconColor}
```

Comment:
The Row component does not need to be passed down any of the setState functions because it is presentational